app/ios/iosutils.cpp

```diff
@@ -29,12 +29,12 @@ bool IosUtils::isIos() const
 
 void IosUtils::callImagePicker( const QString &targetPath )
 {
-  mImagePicker->showImagePicker( 0, targetPath, nullptr );
+  mImagePicker->showImagePicker( 0, targetPath, nullptr, nullptr );
 }
 
 void IosUtils::callCamera( const QString &targetPath )
 {
-  mImagePicker->showImagePicker( 1, targetPath, mPositionKit );
+  mImagePicker->showImagePicker( 1, targetPath, mPositionKit, mCompass );
```

Comment:
remove showImagePicker function and create 2 native functions, one for Camera, one for gallery